pylint/lint/run.py

```diff
@@ -75,7 +75,7 @@ def _cpu_count() -> int:
         cpu_count = multiprocessing.cpu_count()
     else:
         cpu_count = 1
-    if cpu_share is not None:
+    if cpu_share:
```

Comment:
If `cpu_share` is `2/1024` and thus `int(0)` and `cpu_count` is `4` wouldn't it be better to return `1` instead of the `4` we would return with this flow?

Reply:
I didn't grasp exactly how the share/quota etc. works before. Now after reading some stuff about how cpu usage is configured in Docker and Kubernetes, I agree, it should return 1.
I'll rework that again.

Reply:
Changed, but I still need to update my test though.